Alkiiro/refactor-tractor
# Additional parent methods

What's this PR do?
This PR update created the `createPost` and `sendPost` method for activity data to be sent/updated to the API.  Also added code for Activity button to be disabled until all fields have been populated with only numbers.  I also removed Sleep/Activity methods/testing for getWeeklyStats since they have been moved to Repo class.  Small changes were also made to Index.html to make sure input fields can only be entered as numbers.

Where should the reviewer start?
I would start in Index.js reviewing the `getPost` and `sendPost` method.  Then I would take a look at the jQuery `.click` and `.keyup` functions for the activity form.  These disable/enable the button, make sure all fields have been entered, and reset the fields after button is hit.  

How should this be manually tested?
You can fetch the data to see data being added to activity data at API.  You can also test out the button and input fields to make sure they are working correctly on your end.

Any background context you want to provide?
This will greatly enhance our site since now users can update data in activity and implement great functionality for the UI.
